tools/sctool/src/main/java/com/horizen/CommandProcessor.java

```diff
@@ -571,6 +573,10 @@ private void processGenesisInfo(JsonNode json) {
 
             MainchainBlockReference mcRef = MainchainBlockReference.create(mcBlockBytes, params, versionsManager).get();
 
+            // TODO for the time being we have the sc creation only, but we might support also FT
```

Comment:
do we need this todo? I guess ScCr+FT case is supported even now.

Reply:
Removed